pkg/reconciler/dependencybuild/buildrecipeyaml.go

```diff
@@ -516,6 +516,13 @@ func createPipelineSpec(log logr.Logger, tool string, commitTime int64, jbsConfi
 						StringVal: tlsVerify,
 					},
 				},
+				{
+					Name: "JVM_BUILD_SERVICE_DOMAIN_PROXY_IMAGE",
```

Comment:
This would probably be an image providing build enviroment. Meaning all build environment images, must have the domain-proxy client. Or do I miss something ?

Reply:
@matejonnet this PR isn't ready for review. It's still wip. Please ignore it for now, thanks.

Reply:
Still lots of changes to be made.

Reply:
Comment is somewhat outdated now. But yes, the domain proxy will need to be part of the image referenced to by the BUILD_IMAGE param, assuming HERMETIC and ENABLE_DOMAIN_PROXY params are enabled.